### ISSUE TITLE
Cannot access extended class methods 

### ISSUE THREAD

**Issue by luispereira** [urbanairship/android-library#122]
I'm trying to extend the class PublicNotificationExtender, in order to have some custom behaviour into my application with my class, or a wrapper if you prefer on UrbanAirship calls.

Therefore I'm using something like this:
```
class PublicNotificationExtender(context: Context, message: PushMessage) : PublicNotificationExtender(context, message) {
}
```

However, the interesting part is that whenever I initialise my class I cannot access `extend()` method, despite being public

```
  PublicNotificationExtender publicNotificationExtender = new PublicNotificationExtender(context, message);
```

The only way I can do this is to override the class methods, something like this:
```
class PublicNotificationExtender(context: Context, message: PushMessage) : PublicNotificationExtender(context, message) {

    override fun setAccentColor(accentColor: Int): PublicNotificationExtender {
        return super.setAccentColor(accentColor)
    }

    override fun setSmallIcon(smallIcon: Int): PublicNotificationExtender {
        return super.setSmallIcon(smallIcon)
    }

    override fun extend(builder: NotificationCompat.Builder?): NotificationCompat.Builder {
        return super.extend(builder)
    }

    override fun setLargeIcon(largeIcon: Int): PublicNotificationExtender {
        return super.setLargeIcon(largeIcon)
    }
}
```

Do you guys have annotation processor hiding these methods? What am I missing here?

**Comment by rlepinski** [comment]
Nothing is standing out to me on why it would be hidden. What SDK version are you using?

**Comment by rlepinski** [comment]
Just tried it in a sample project with SDK 10 and it works fine. Could you try giving your class a custom name such as `Custom PublicNotificationExtender` ?

**Comment by luispereira** [comment]
I'm using UrbanAirship 9.7.1.
I tried with a different class name and it still does not work. I also tried putting the class in java, and still no success.

I'm still investigating the issue, however I think the issue is related with my application, which only imports the module which wrapper the urban airship, meaning that I do not have access to the UrbanAirship classes, which kinda makes sense 🤔
